setup.py

```diff
@@ -75,7 +75,10 @@
                               "Routes>=2.0",
                               "backlash>=0.0.5",
                               "Shapely>=1.2",
-                              "GeoAlchemy2>=0.2.4"],
+                              "GeoAlchemy2>=0.2.4",
+                              "ipython<=5.3.0" if sys.version_info < (3, 3) else "ipython>=6.0.0"],
```

Comment:
This conditional must be computed once and passed via variable; alternatively, you may wrap it into lambda. Copy-pasted code snippets lead to unexpected diversity increase of code base over time.